### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by AlyoshaVasilieva** [argotronic/argus_data_api#2]
Is this code available under any certain license? Argus Monitor is of course proprietary, but if I write some tool that uses this API (using the headers) can I release it under an open source license?

(For example: a tool that takes the data from the API and sends it to a microcontroller which will show it on an attached display)

**Comment by argotronic** [comment]
Yes, I forgot to add any license information. I will have to look into that, but in any case, you will be able to use the code in this repository without limitation (even in closed source and sell it if you desire ;) ). 

So, this will most likely go under MIT license, but is not an official statement as I would have to do some research which license fits. The statement above still stands and you can basically do with the software provided in this repo whatever you want.
